dll/cpl/desk/lang/ro-RO.rc

```diff
@@ -1,5 +1,6 @@
 /*
- * Translators: Petru Dimitriu (petrimetri at gmail dot com)
+ * Translators: Andrei Miloiu
```

Comment:
When adding your name here, then please also add an email address for others to contact you. Otherwise it is rather useless to add your name here. Furthermore I do recommend to add yourself at the bottom, since you did not create the file.